StreamChatClient_v3/WebSocketClient/Events/EventPayload.swift

```diff
@@ -100,14 +88,33 @@ struct EventPayload<ExtraData: ExtraDataTypes>: Decodable {
         self.message = message
         self.reaction = reaction
         self.watcherCount = watcherCount
-        self.unreadChannelsCount = unreadChannelsCount
-        self.unreadMessagesCount = unreadMessagesCount
+        self.unreadCount = unreadCount
         self.createdAt = createdAt
         self.isChannelHistoryCleared = isChannelHistoryCleared
         self.banReason = banReason
         self.banExpiredAt = banExpiredAt
     }
     
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        eventType = try container.decode(EventType.self, forKey: .eventType)
+        connectionId = try container.decodeIfPresent(String.self, forKey: .connectionId)
+        cid = try container.decodeIfPresent(ChannelId.self, forKey: .cid)
+        currentUser = try container.decodeIfPresent(CurrentUserPayload<ExtraData.User>.self, forKey: .currentUser)
+        user = try container.decodeIfPresent(UserPayload<ExtraData.User>.self, forKey: .user)
+        createdBy = try container.decodeIfPresent(UserPayload<ExtraData.User>.self, forKey: .createdBy)
+        memberContainer = try container.decodeIfPresent(MemberContainerPayload<ExtraData.User>.self, forKey: .memberContainer)
+        channel = try container.decodeIfPresent(ChannelDetailPayload<ExtraData>.self, forKey: .channel)
+        message = try container.decodeIfPresent(MessagePayload<ExtraData>.self, forKey: .message)
+        reaction = try container.decodeIfPresent(ReactionPayload.self, forKey: .reaction)
+        watcherCount = try container.decodeIfPresent(Int.self, forKey: .watcherCount)
+        unreadCount = try? UnreadCount(from: decoder)
```

Comment:
Will this report parsing error correctly when `UnreadCount` will have an incorrect JSON format?

Reply:
Yes, if it present, but incorrect.

Reply:
Actually no, but I don't know how to implement it optionally

Reply:
Let's leave it

Reply:
I found that unread count is always presented in events at least with sample JSON data that I have in tests.